src/glyphs.js

```diff
@@ -111,6 +135,61 @@ const updateMessages = (id, path, req) => {
   )
 };
 
+const importMessages = (line) => {
+  const message = line.split(",");
+  const viewMessage = (message[1] === "Intro");
+
+  let path = "";
+  let newMessage = {"text": message[2]};
+
+  if (viewMessage) newMessage.probability = Number(message[3]);
+  else newMessage.probability = message.splice(3, 5).map(num => Number(num));
+
+  if (viewMessage) path = "view";
+  else path = `view.gauges.${message[1]}`;
+
+  return db.collection.updateOne(
+    {
+      "view.name": message[0].toLowerCase()
+    },
+    {
+      $addToSet: {
+        [`${path}.messages`]: newMessage
+      }
+    }
+  )
+}
+
+const overwriteMessages = (file) => {
+  let first = true;
```

Comment:
Maybe it would be easy and worthwhile to implement another POST field option that would control whether we do this or not?  I just feel like we may want to skip headers sometimes but not at other times.